Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -3491,6 +3492,39 @@ RefPtr<CSSStyleValue> Parser::parse_color_function(TokenStream<ComponentValue>&
         alpha.release_nonnull());
 }
 
+// https://drafts.csswg.org/css-color-5/#funcdef-light-dark
+RefPtr<CSSStyleValue> Parser::parse_light_dark_color_value(TokenStream<ComponentValue>& outer_tokens)
+{
+    auto transaction = outer_tokens.begin_transaction();
+    outer_tokens.discard_whitespace();
+
+    auto const& function_token = outer_tokens.consume_a_token();
+    if (!function_token.is_function("light-dark"sv))
+        return {};
+
+    auto inner_tokens = TokenStream { function_token.function().value };
+    inner_tokens.discard_whitespace();
+
+    auto light = parse_color_value(inner_tokens);
+    if (!light)
+        return {};
+    inner_tokens.discard_whitespace();
+
+    if (!inner_tokens.consume_a_token().is(Token::Type::Comma))
+        return {};
+
+    auto dark = parse_color_value(inner_tokens);
+    if (!dark)
+        return {};
+    inner_tokens.discard_whitespace();
+
+    if (inner_tokens.has_next_token())
+        return {};
+
+    transaction.commit();
+    return CSSLightDark::create(light, dark);
```

Comment:
```suggestion
    return CSSLightDark::create(light.release_nonnull(), dark.release_nonnull());
```
(See comments on `CSSLightDark.h`)